install_template/templates/products/edb-postgres-advanced-server/v15_index.njk

```diff
@@ -0,0 +1,8 @@
+{% extends "./index.njk" %}
```

Comment:
@josh-heyer, we added this file, but it is not getting deployed.

Reply:
The indentation was different between the redirects entries in install.njk and those in v15_install.njk - this resulted in unparsable yaml and the entire file was skipped. I've fixed the indentation, and also updated the deploy script to produce more visible and (slightly) more informative error messages when this happens.